From 58569c87f6c7cfae19a0f67b3445f902d2b19810 Mon Sep 17 00:00:00 2001
From: nikhilad <nikhilad@ami.com>
Date: Sat, 7 Dec 2024 14:05:59 +0530
Subject: [PATCH 1/1] GPIO Pin Configurations for Power Operations

Added the GPIO Pin Configurations for Power Operations
in aspeed-ast2600-evb.dts for EVB-AST2600 Platform

Signed-off-by: nikhilad <nikhilad@ami.com>
---
 .../boot/dts/aspeed/aspeed-ast2600-evb.dts    | 32 +++++++++++++++++++
 1 file changed, 32 insertions(+)

diff --git a/arch/arm/boot/dts/aspeed/aspeed-ast2600-evb.dts b/arch/arm/boot/dts/aspeed/aspeed-ast2600-evb.dts
index 016bca87254d..2d33964a69fb 100644
--- a/arch/arm/boot/dts/aspeed/aspeed-ast2600-evb.dts
+++ b/arch/arm/boot/dts/aspeed/aspeed-ast2600-evb.dts
@@ -463,6 +463,38 @@ &sgpiom0 {
 	status = "okay";
 	ngpios = <128>;
 };
+
+&gpio0 {
+    status = "okay";
+    gpio-line-names =
+    /*A0-A7*/       "","","","","","","","",
+    /*B0-B7*/       "","","","","","","","",
+    /*C0-C7*/       "","","","","","","","",
+    /*D0-D7*/       "","","","","","","","",
+    /*E0-E7*/       "","","","","","","","",
+    /*F0-F7*/       "","","","","","","","",
+    /*G0-G7*/       "","","","","","","","",
+    /*H0-H7*/       "","","","","","","","",
+    /*I0-I7*/       "","","","","","","","",
+    /*J0-J7*/       "","","","","","","","",
+    /*K0-K7*/       "","","","","","","","",
+    /*L0-L7*/       "","","","","","","","",
+    /*M0-M7*/       "","","","","","","","",
+    /*N0-N7*/       "","","","","","","","",
+    /*O0-O7*/       "","","","","","","","",
+    /*P0-P7*/       "","","","","","","","",
+    /*Q0-Q7*/       "","","","","","","","",
+    /*R0-R7*/       "","","","","","","","",
+    /*S0-S7*/       "","","","","","","","",
+    /*T0-T7*/       "","","","","","","","",
+    /*U0-U7*/       "","","","","","","","",
+    /*V0-V7*/       "NMI_OUT","RESET_BUTTON","NMI_BUTTON","POWER_BUTTON","PS_PWROK","POST_COMPLETE","POWER_OUT","RESET_OUT",
+    /*W0-W7*/       "ID_BUTTON","SIO_S5","SIO_ONCONTROL","","","","","",
+    /*X0-X7*/       "","","","","","","","",
+    /*Y0-Y7*/       "","","","","","","","",
+    /*Z0-Z7*/       "","","","","","","","";
+};
+
 &kcs1 {
         aspeed,lpc-io-reg = <0xCA0>;
         status = "okay";
-- 
2.25.1

